cluster.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const process = require('process');
```

Comment:
is this required?

Reply:
Yes on last line we are listening on process `uncaughtException`

Reply:
`process` is global AFAIK. this require seems redundant

Reply:
ahh eslint is complaining after removing process have updated the es lint also for allowing global vars

Reply:
updated